examples/v1/pipelineruns/beta/7392-regression.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1
```

Comment:
Just double-checking this is created to be run in beta integration tests to make sure future changes will not introduce regression bugs as reporeted in #7392?

Reply:
Yes, since array results is still beta